Makefile

```diff
@@ -12,7 +12,7 @@ endif
 BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMG ?= meshery-meshsync:latest
```

Comment:
Shouldn't it be meshery-operator.  As meshery-meshsync is for meshsync controller.

Reply:
yep, my mistake